src/api/txMining.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import txMiningRequestClient from './txMiningAxios';
+
+/**
+ * Api calls for tx mining
+ *
+ * @namespace ApiTxMining
+ */
+
+const txMiningApi = {
+  /**
+   * Submit tx to be mined
+   *
+   * @param {String} tx data in hexadecimal
+   *
+   * @return {Promise}
+   * @memberof txMiningApi
+   * @inner
+   */
+  submitJob(tx, propagate, add_parents, resolve) {
+    const postData = {tx, propagate, add_parents};
+    return txMiningRequestClient(resolve).post('submit-job', postData).then((res) => {
+      resolve(res.data)
+    }, (res) => {
+      return Promise.reject(res);
+    });
+  },
+
+  /**
+   * Get job status
+   *
+   * @param {String} job Job id
+   *
+   * @return {Promise}
+   * @memberof txMiningApi
+   * @inner
+   */
+  getJobStatus(job, resolve) {
+    const data = {'job-id': job};
```

Comment:
We probably want to have a standard for json naming. I'd say we should use `job_id` here to follow what we've been using.

Reply:
Actually I am following what we've been using in the new events in the classes. Check `src/new/wallet.js` and all new events are like `job-id`. We can decide to change but I think it's better to keep like this now, since it's our standard in the lib.

Reply:
But this is not an event, this is the API parameter.

I mixed things up, it's not json as it's a get request. Anyway, I think we've been using `_` on all requests and `-` for local events.

Reply:
Ahhh, my bad! I thought this was another file, sorry.

Actually this `job-id` was 'defined' in the design https://github.com/HathorNetwork/rfcs/issues/24 (it does not say that the GET parameter is this, but almost). And this is what @msbrogli has been using in his code, I am just expecting here what he is sending there.

This code won't be merged before his PR, so I will keep this conversation opened, so we can finish this discussion in his PR (and I change here, if needed).

Reply:
Got it, thanks.

Reply:
I implemented following the design in the `tx-mining-service` as well.

Reply:
This is something I didn't notice when reviewing the design, but I think it makes sense to stick to one "convention". We've been using mostly `_`, so I think we should continue.

Reply:
I agree that we should stick to a convention, however we are already using more than one, e.g. https://api.economics.hathor.network/circulating-supply.

So we should first define which convention we will use. I think we should change the endpoints on the tx-mining-api if we decide to use `_`

Reply:
Ok, agreed